package-lock.json

```diff
@@ -2728,9 +2728,9 @@
       "integrity": "sha1-2cjt3h2nnRJaFRt5UzuXhnY0auU=",
       "dev": true,
       "requires": {
-        "glob": "~3.1.21",
-        "lodash": "~1.0.1",
-        "minimatch": "~0.2.11"
+        "glob": "3.1.21",
```

Comment:
🤔 `glob` has changed from `~` version to an exact version. Do we know why this has happened?